src/tasks/helpers/getExtensionDescriptors.js

```diff
@@ -27,7 +27,10 @@ module.exports = () => {
 
     const extensionDescriptor = require(resolvedExtensionDescriptorPath);
     extensionDescriptor.extensionDescriptorPath = resolvedExtensionDescriptorPath;
-    extensionDescriptors[extensionDescriptor.name] = extensionDescriptor;
+
+    if (platform === extensionDescriptor.platform) {
```

Comment:
@dompuiu What is the consequence of having no name here if the platform doesn't match?

Reply:
@brenthosie You will always have a sandbox extension and the extension you are developing.